src/webui/api/appcontroller.cpp

```diff
@@ -281,8 +280,6 @@ void AppController::preferencesAction()
     data["disk_cache_ttl"] = session->diskCacheTTL();
     // Enable OS cache
     data["enable_os_cache"] = session->useOSCache();
-    // Guided read cache
-    data["enable_guided_read_cache"] = session->isGuidedReadCacheEnabled();
```

Comment:
you should also remove the relevant entries in html/WebUI.

Reply:
Ok, removed.